pkgs/by-name/aw/aws-azure-login/package.nix

```diff
@@ -1,13 +1,15 @@
 { lib
 , stdenv
+, callPackage
+, chromium
 , fetchFromGitHub
 , fetchYarnDeps
 , makeWrapper
 , nodejs
+, pkgs
```

Comment:
Don't import all `pkgs` for using `pkgs.runCommand`. Just import `runCommand` instead .